src/components/content/monitor/MonitorChart.tsx

```diff
@@ -145,7 +154,7 @@ export const MonitorChart = ({
     const startFetchMonitorMetricDataTimer = () => {
         try {
             refreshMonitorMetrics.current = window.setInterval(() => {
-                getMetricResponse(serviceId, activeMonitorMetricType as Metric.monitorResourceType, false);
+                void monitorMetricQuery.refetch();
```

Comment:
@Alice1319 instead of we refetching manually, can you see if we can use the "refetchInterval" from react-query directly?
you can see an example here? https://github.com/eclipse-xpanse/xpanse-ui/blob/main/src/components/content/order/OrderSubmitStatusPolling.tsx#L31

Reply:
fixed